add_mozilla_tag.py

```diff
@@ -0,0 +1,11 @@
+from pelican import signals
+
+def add_category(generator):
+    for article in generator.articles:
+        if hasattr(article, 'tags'):
+            article.tags.append('mozilla')
+        else:
+            setattr(article, 'tags', ['mozilla'])
```

Comment:
Yeah, I know, this is due to [how pelican handles metadata](https://github.com/getpelican/pelican/blob/master/pelican/contents.py#L62-L65) (and the worse here is that **I** wrote this code…)